### PR TITLE
Added default new issue and pull request templates for the branch

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
This is done to reduce contributions to this branch as much as possible.

### CODE REVIEW

File: .github/ISSUE_TEMPALTE.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This repo is OBSOLETE - please see the README file for information
```
Comment (Eilon) [inline .github/ISSUE_TEMPALTE.md]: Do you plan to update the README as well? If so, can you include it in the PR?

Comment (mkArtakMSFT) [reply .github/ISSUE_TEMPALTE.md]: Updated the text in Readme and added @danroth27 as a reviewer.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
-ASP.NET Core MVC Precompilation
+ ASP.NET Core MVC Precompilation (obsolete)
 ===
 
+## Obsolete
+We are planning to release a new feature from the aspnet/Razor repo, which would cover the scenarios provided by the tools in this repo. As a result, this repo is now obsolete. If you'd like to learn more about the new feature we will be replacing this with, please look [here](https://github.com/aspnet/Razor/issues/1740).
```
Comment (Eilon) [inline README.md]: Can make the `aspnet/Razor` text by a link.

**Comment by pranavkm** [comment]
Follow up to this - are we ready to stop building this repo in Universe?

**Comment by Eilon** [comment]
@pranavkm - hmm, is this package part of the metapackage? I don't think we can stop building this or shipping it at least until 3.0...

**Comment by mkArtakMSFT** [comment]
@danroth27, I'm waiting for your signoff.

**Comment by pranavkm** [comment]
@Eilon  - yup, it is: https://www.nuget.org/packages/Microsoft.AspNetCore.All. We probably have to keep the target file around and have it no-op \ warn instead. Does that sound reasonable?

File: .github/ISSUE_TEMPALTE.md (hunk 1 above)
Comment (danroth27) [reply .github/ISSUE_TEMPALTE.md]: I worry that saying the repo is obsolete will make users think that the feature is already obsolete, even though we haven't shipped its replacement yet.

How about: "NOTE: Future enhancements for Razor compilation should now be opened in the [Razor](https://github.com/aspnet/razor) repo. Only issues (bugs) with the existing MVC precompilation feature should be opened here"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1 @@
+This repo is OBSOLETE - please see the README file for information
```
Comment (danroth27) [inline .github/PULL_REQUEST_TEMPLATE.md]: Same as above

File: README.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
-ASP.NET Core MVC Precompilation
+ ASP.NET Core MVC Precompilation (obsolete)
 ===
 
+## Obsolete
```
Comment (danroth27) [inline README.md]: How about we remove "Obsolete" and replace the text with the following note?:

"NOTE: This repo is solely for maintenance of the existing MVC precompilation feature. Future work on Razor compilation is now being handled in the [Razor](https://github.com/aspnet/razor repo. See https://github.com/aspnet/Razor/issues/1740 for additional details."

Comment (mkArtakMSFT) [reply README.md]: @Eilon, any objections? If not, I'll just use @danroth27's suggestion.

Comment (Eilon) [reply README.md]: Yeah this sounds good to me.

**Comment by Eilon** [comment]
@pranavkm if this still ships in the metapackage then we can't stop building it in Universe, right? Or did I misunderstand something?

**Comment by mkArtakMSFT** [comment]
@danroth27, the changes are made as you've requested. Can you please sign off?

**Comment by pranavkm** [comment]
> @pranavkm if this still ships in the metapackage then we can't stop building it in Universe, right? Or did I misunderstand something?

Right. That said, we would want to keep the API surface i.e. MSBuild targets, but have them no-op \ tell you to use Razor.Design or something along those lines. My guess is we wouldn't want to continue running the code from this repo, no?

**Pull request merged**